### PR TITLE
Update DG to include implementation of `Copy` feature

### PR DESCRIPTION
**Pull request by minosx31**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -279,6 +279,19 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 *{Explain here how the data archiving feature will be implemented}*
 
+### Copy to clipboard
```
Comment (rexcyrio) [inline docs/DeveloperGuide.md:282]: Is the table of contents updated?

Comment (minosx31) [reply docs/DeveloperGuide.md]: Ah missed that out

**Review by rexcyrio — APPROVED** [review]
LGTM :+1:

File: docs/images/CopySequenceDiagram.png
Comment (niekis) [inline docs/images/CopySequenceDiagram.png:1]: when `CopyCommand` returns to `AddressBookParser`, I think you're missing a 'c' label on the return arrow.

Comment (niekis) [inline docs/images/CopySequenceDiagram.png:1]: You could also add a 'result' label on the return arrow from `CopyCommand` to `LogicManager`, like in the `delete` diagram

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -275,6 +277,24 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 *{more aspects and alternatives to be added}*
 
+### Copy feature
+
+#### Current Implementation
+
+The copy feature is implemented by extracting information of the specified `Person` and then setting it as the content of the user's system's clipboard. The copy mechanism is facilitated by `CopyCommand` which extends `Command`.
```
Comment (niekis) [inline docs/DeveloperGuide.md:284]: Not sure if it's relevant, but you could briefly explain how you copied it to clipboard?

**Review by niekis — CHANGES_REQUESTED** [review]
looks pretty good, just some minor comments that could make our DG look more stanardized

**Comment by rexcyrio** [comment]
@niekis I think you can open a new issue with label `type.Bug` that references these comments so that @minosx31 can recitfy them properly (I can't find a way to re-open a PR that has already been merged lol)

**Pull request merged**